### PR TITLE
Reuse test execution for Eventing from knative-eventing-src

### PR DESCRIPTION
**Pull request by mgencur**
Do not duplicate code for test execution but reuse what is already in knative-eventing.

### CODE REVIEW

File: test/eventing.bash [hunk 1]
```diff
@@ -2,23 +2,18 @@
...
-  oc patch cm config-br-defaults -n knative-eventing -p '{"data":{"default-br-config":"clusterDefault:\n  brokerClass: MTChannelBasedBroker\n  apiVersion: v1\n  kind: ConfigMap\n  name: config-br-default-channel\n  namespace: knative-eventing\n"}}' --type=merge || failed=2
+  cd "$KNATIVE_EVENTING_HOME" || return $?
 
-  go_test_e2e -timeout=90m -parallel=12 ./test/e2e -brokerclass=MTChannelBasedBroker -channels=messaging.knative.dev/v1alpha1:InMemoryChannel,messaging.knative.dev/v1alpha1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel \
-    --kubeconfig "$KUBECONFIG" \
-    --imagetemplate "$image_template" || failed=3
+  source "${KNATIVE_EVENTING_HOME}/openshift/e2e-common.sh"
 
+  # run_e2e_tests defined in knative-eventing
+  run_e2e_tests || failed=1
```
Comment (matzew) [inline test/eventing.bash:16]: will that work w/ all the `failed` etc things ?

Comment (mgencur) [reply test/eventing.bash:16]: You mean it should be `run_e2e_tests || failed=$?`

Comment (mgencur) [reply test/eventing.bash:16]: `failed` is a local variable, and we also run in a subshell (see the surrounding brackets). I don't understad what you mean.

**Pull request merged**